packages/vite/src/node/plugins/css.ts

```diff
@@ -312,6 +317,15 @@ export function cssPostPlugin(config: ResolvedConfig): Plugin {
       // build CSS handling ----------------------------------------------------
 
       // record css
+      // cache css compile result to map
+      if (inlined && isHTMLProxy) {
+        addToHTMLProxyTransformResult(
+          `${cleanUrl(id)}_${Number.parseInt(query!.index)}`,
+          css
+        )
+        // and replace inline style when `renderChunk` in vite:html-inline-proxy plugin
```

Comment:
Should this be when `generateBundle` in `vite:build-html` plugin?

Reply:
we need cache the css transform result. and need return emtry string for the expression `import 'index.html?inline-css&html-proxy&index=1.css'`

Reply:
I'm confused. Perhaps I didn't quote my initial comment correctly. I meant should the comment be "and replace inline style when `generateBundle` in `vite:build-html` plugin"? I don't see a `renderChunk` hook in the `vite:html-inline-proxy` plugin.

Reply:
sorry, I thought you said  need make the logic to inline-html plugin😂